digiwf-apps/packages/apps/digiwf-tasklist/src/views/ProcessInstances.vue

```diff
@@ -163,5 +192,68 @@ export default class ProcessInstances extends Vue {
     return this.processInstances.filter(task => JSON.stringify(Object.values(task)).toLocaleLowerCase().includes(this.filter.toLocaleLowerCase()));
   }
 
+  get isFilterPersistent(): boolean {
+    if (
+      !this.filter ||
+      this.filter.length == 0 ||
+      !this.persistentFilters ||
+      this.persistentFilters!.length == 0
+    ) {
+      return false;
+    }
+    return (
+      this.persistentFilters!.find(
+        (fl: FilterTO) => fl.filterString == this.filter
+      ) != undefined
+    );
+  }
+
+  async savePersistentFilter() {
+    if (!this.filter){
+      return;
+    }
+    const request: SaveFilterTO = {
+      pageId: "processinstances",
+      filterString: this.filter,
+    }
+    try {
+      const cfg = ApiConfig.getAxiosConfig(FetchUtils.getPUTConfig({}));
+      await FilterRestControllerApiFactory(cfg).saveFilter(request);
+
+      this.errorMessage = "";
+      this.$store.dispatch('filters/getFilters', true);
+    } catch (error) {
+      this.errorMessage = 'Der Filter konnte nicht gespeichert werden.';
+    }
+  }
+
+  async deletePersistentFilter() {
+    const id = this.persistentFilters!.find((f: FilterTO) => f.filterString == this.filter)?.id!
+    try {
+      const cfg = ApiConfig.getAxiosConfig(FetchUtils.getDELETEConfig());
+      await FilterRestControllerApiFactory(cfg)._delete(id);
+
+      this.errorMessage = "";
+      this.$store.dispatch('filters/getFilters', true);
+    } catch (error) {
+      this.errorMessage = 'Der Filter konnte nicht gelöscht werden.';
+    }
+  }
+
+  async loadPersistentFilters(refresh = false): Promise<void> {
+    this.persistentFilters = this.$store.getters['filters/filters'].filter((filter: FilterTO) => filter.pageId === "processinstances");
+    try {
+      await this.$store.dispatch('filters/getFilters', refresh);
+      this.errorMessage = "";
+    } catch (error) {
+      this.errorMessage = error.message;
+    }
```

Comment:
The code block with the api calls is exactly the same as in tasklist.vue. Should we move this into a seperate component or function?

Reply:
I thought about that. But the only solution at the time is a mixin, which isnt supported by composition api anymore. I recommend waiting until we switched all sources to composition api and then extract the api call logic to the composition variant of a mixin. Ive added an issue for that: https://github.com/it-at-m/digiwf-core/issues/46

Reply:
Sounds good to me. Thanks for opening the issue